apps/utils/scripts/README.md

```diff
@@ -0,0 +1,440 @@
+# Utility Scripts
+
+The scripts in this directory are intended to simplify building and running BFD on your local system.
+The scripts are:
+
+* `run-bfd-pipeline`: Runs the latest version of the BFD ETL pipeline in either RIF or RDA mode.
+* `run-bfd-server`: Runs the latest version of the BFD FHIR server.
+* `run-db-migrator`: Runs the latest version of the BFD database migrator.
+* `run-rda-server`: Runs a mock RDA API server in one of several modes.
+* `build-bfd`: Simplifies running maven to build BFD with a variety of options.
+* `create-bfd-db`: Create a new postgresql 14 database running in a docker container.
+
+# Dependencies
+
+These scripts are intended to be as generic as possible while still accomplishing their task.
+They have been developed and tested on a Macbook running Ventura.
+
+* All of the scripts are written in `bash` and assume it is runnable as `/bin/bash`.
+* Several of the scripts assume the presence of `lsof` in the user's PATH. This may not be available on linux or may use
+  different command line options.
+* Several of the scripts assume the presence of a `docker` command that can be run without sudo.
+* Most of the scripts require the presence of a Java JDK version 17 or greater.
+* Most of the scripts require the `BFD_PATH` and `BFD_EXEC` environment variables. See below for details on those.
+
+# Run Scripts
+
+## Overview
+
+All of the run scripts operate in the same basic way. Each of them requires two environment variables
+to be set in your local environment.
+
+* `BFD_PATH`: The path to the BFD source repository. For example `$HOME/projects/beneficiary-fhir-data`.
+* `BFD_EXEC`: The path to a directory that all of the run scripts can use to store files needed to run the programs.
+  The `BFD_EXEC` directory should be a stand-alone, permanent directory outside of the BFD repository.
+  For example `$HOME/bfd-exec`.
+
+In addition the scripts assume they can find your maven artifact repository under `$HOME/.m2/repository` and your maven
+build cache
+under `$HOME/.m2/cache`.
+
+Each of the run scripts will create an app specific working directory under `$BFD_EXEC`. The scripts all have the same
+basic operating mode:
+
+* Verify that `BFD_PATH` and `BFD_EXEC` are defined and that directories exist as expected.
```

Comment:
Think it will be helpful if we add the `cmds` to set the Environments variables in the README.md